libr/util/p_format.c

```diff
@@ -1219,7 +1219,7 @@ int r_print_format_struct_size(const char *f, RPrint *p, int mode, int n) {
 	if (!f) {
 		return -1;
 	}
-	if (n >= 3) {
+	if (n >= 5) {  // This is the nesting level, is this not a bit arbitrary?!
```

Comment:
I think we may change it to the timeout instead.

Reply:
Are we addressing this issue in this PR, if so what solution are we thinking is best? I think config var with a sane default is probably the best approach.